config/config/sc-config.yaml

```diff
@@ -354,6 +354,9 @@ harbor:
   backup:
     enabled: true
     retentionDays: 7
```

Comment:
Is the retention on disc(volume) the same as in S3 ? 7 days as shown here?
If yes, where is the part that cleans up the disc(volume) ? Or the volume is being deleted after the completion of the backup job and then recreated each time a new backup job is running?

Reply:
The disk is ephemeral, so it's deleted when the job is done. It's just so it doesn't get evicted if you run this job on nodes that doesn't have that much storage.

Reply:
I missed paying attention to the volume type being ephemeral. Sorry
LGTM :)